src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue23399.cs

```diff
@@ -0,0 +1,38 @@
+#if ANDROID
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue23399(TestDevice device) : _IssuesUITest(device)
+	{
+		public override string Issue => "Closing Modal While App is Backgrounded Fails"; 
+
+		[Test]
+		public void MakingFragmentRelatedChangesWhileAppIsBackgroundedFails()
+		{
+			try
+			{
+				App.WaitForElement("OpenModal");
+				App.Tap("OpenModal");
+				App.WaitForElement("StartCloseModal");
+				App.Tap("StartCloseModal");
+				App.BackgroundApp();
+				App.WaitForNoElement("StartCloseModal");
+				App.ForegroundApp();
+				App.WaitForElement("OpenModal");
+			}
+			catch
```

Comment:
We don't need these bits anymore

We had to do this before we were just restarting the app between every test

So, you're fine to just remove this try/catch